org.civicrm.bootstrapcivicrm/js/civihr-popup.js

```diff
@@ -0,0 +1,62 @@
+/*
+Prevent Popups to overflow tables.
+*/
+CRM.$(function($) {
+  'use strict';
+
+  (function($) {
+    var MutationObserver = window.MutationObserver || window.WebKitMutationObserver || window.MozMutationObserver;
+
+    $.fn.attrchange = function(callback) {
+      if (MutationObserver) {
+        var options = {
+          subtree: false,
+          attributes: true
+        };
+
+        var observer = new MutationObserver(function(mutations) {
+          mutations.forEach(function(e) {
+            callback.call(e.target, e);
+          });
+        });
+
+        return this.each(function() {
+          observer.observe(this, options);
+        });
+      }
+    }
+  })($);
+
+  // .btn-slide is a continer of popup and a button triggering it
+  $('.btn-slide').attrchange(function(e) {
+    var $button = $(this),
+        $popup,
+        button_offset;
+
+    if($button.hasClass('btn-slide-active')) {
+      // check if popup is open
+      $popup = $button.children('ul.panel');
+
+      $popup.appendTo('body')
+        .addClass('civihr-popup');
+
+      button_offset = $button.offset();
+
+      $popup.css('position', 'absolute')
```

Comment:
It seems that this could be handled by css better.

You could either write a css selector like
```css
body > {popup class} { position: absolute; }
```
or use the `.civihr-custom` class not for styling it (see my previous comment), but just for applying the `position` property